package.json

```diff
@@ -60,7 +60,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "pretty-quick --staged --pattern 'packages/**/src/**/*' && npx eslint . --ext .ts"
+      "pre-commit": "pretty-quick --staged --pattern 'packages/api/src/**/*' && npx eslint . --ext .ts"
```

Comment:
excluding types from pretty-quick as auto generated types now contain new syntax which is 
import type { Vec, u16, u32, u64 } from '@polkadot/types';
which makes prettier unhappy i,e
/Users/karishma/work/test/api.js/node_modules/prettier/index.js:9750
      throw error;
      ^

SyntaxError: '=' expected. (4:13)
  2 | /* eslint-disable */
  3 | 
> 4 | import type { u256 } from '@polkadot/types';
    |             ^
  5 |

Reply:
It makes sense to exclude the auto-generated code from such checks.